go/tools/builders/cgo.go

```diff
@@ -132,7 +135,15 @@ func run(args []string) error {
 			if err := ioutil.WriteFile(cOut, []byte(""), 0644); err != nil {
 				return err
 			}
-		} else if isCgo {
+			continue
+		}
+
+		if pkgName != "" && pkg != pkgName {
```

Comment:
I'm not sure there is a definite right answer here, but I don't think files that were filtered out should cause this error.